core/src/apps/ethereum/sign_tx.py

```diff
@@ -17,7 +17,7 @@
 )
 
 # maximum supported chain id
-MAX_CHAIN_ID = 2147483629
+MAX_CHAIN_ID = 9223372036854775790
```

Comment:
same as C version, this should be a calculation:
```
# maximum supported chain id, so that signature_v fits into an uint64
MAX_CHAIN_ID = (0xFFFF_FFFF_FFFF_FFFF - 35) // 2
```

Reply:
Done, fixed